_providers/t-online.md

```diff
@@ -13,6 +13,7 @@ servers:
     socket: SSL
     hostname: securesmtp.t-online.de
     port: 465
+before_login_hint: To use Delta Chat with a T-Online email address, you need an app password.
```

Comment:
Good catch, I forgot the hint.

This could be a bit more verbose on what to do next though, what about "To use Delta Chat with a T-Online email address, you need to create an app password in the web interface."?